packages/@react-types/calendar/src/index.d.ts

```diff
@@ -65,15 +65,15 @@ export interface RangeCalendarProps<T extends DateValue> extends CalendarPropsBa
   allowsNonContiguousRanges?: boolean
 }
 
-export interface SpectrumCalendarProps<T extends DateValue> extends CalendarProps<T>, DOMProps, StyleProps {
+export interface SpectrumCalendarProps<T extends DateValue> extends CalendarProps<T>, DOMProps, StyleProps, AriaLabelingProps {
```

Comment:
In the same vein as the previous comments, should these `AriaLabelingProps` go onto CalendarPropsBase or CalendarProps instead so they show up for the aria hooks? They currently don't exist in the useCalendar props dropdown in the docs. To follow our previous conventions we could also create `AriaCalendarProps` and `AriaRangeCalendarProps` and supply those to the aria hooks as well